zwave_js_server/model/value.py

```diff
@@ -42,17 +42,31 @@ class ValueDataType(TypedDict, total=False):
     ccVersion: int
 
 
-def get_value_id(node: "Node", event_data: ValueDataType) -> str:
-    """Return ID of value."""
-    command_class = event_data["commandClass"]
-    endpoint = event_data.get("endpoint") or "00"
-    property_ = event_data["property"]
-    # Some values that have a property key don't have a property key name, so we will
-    # fall back to property key if it exists
-    property_key = (
-        event_data.get("propertyKeyName") or event_data.get("propertyKey") or "00"
+def get_value_id_from_dict(node: "Node", val: ValueDataType):
```

Comment:
Do we want to expose this helper or keep it private?

Reply:
We can keep it private since it's only useful for receiving data from the server